kiali/README.md

```diff
@@ -0,0 +1,85 @@
+# Install custom Kiali in Kyma
+
+## Overview
+
+The Kyma Kiali component brings limited configuration options in contrast to the upstream [`kiali-operator`](https://github.com/kiali/helm-charts/tree/master/kiali-operator) chart. Modifications might be reset at next upgrade cycle.
+
+An alternative can be a parallel installation of the upstream chart offering all customization options. This tutorial outlines how to achieve such installation in co-existent to the Kyma stack.
+
+## Prerequisites
+
+- Kyma as the target deployment environment.
+- kubectl > 1.22.x
+- helm 3.x
+
+## Installation
+
+### Preparation
+
+1. Export your Namespace as a variable. Replace the `{namespace}` placeholder in the following command and run it:
+
+    ```bash
+    export KYMA_EXAMPLE_NS="{namespace}"
+    ```
+
+1. Export the Helm release name which you want to use. It can be any name, be aware that all resources in the cluster will be prefixed with that name. Replace the `{release-name}` placeholder in the following command and run it:
```

Comment:
saying any name can be confusing as user might (in rare case) reuse some existing release_name. May be we can warn about this ?